DocSum/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -120,47 +148,65 @@ docker compose up -d
      -H 'Content-Type: application/json'
    ```
 
-3. MegaService
+3. Whisper Microservice
 
    ```bash
-   curl http://${host_ip}:8888/v1/docsum \
-       -H "Content-Type: multipart/form-data" \
-       -F "messages=Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5." \
-       -F "max_tokens=32" \
-       -F "language=en" \
-       -F "stream=false"
+    curl http://${host_ip}:7066/v1/asr \
+        -X POST \
+        -d '{"audio":"UklGRigAAABXQVZFZm10IBIAAAABAAEARKwAAIhYAQACABAAAABkYXRhAgAAAAEA"}' \
+        -H 'Content-Type: application/json'
    ```
 
-Following the validation of all aforementioned microservices, we are now prepared to construct a mega-service.
+   Expected output:
 
-## 🚀 Launch the UI
+   ```bash
+     {"asr_result":"you"}
+   ```
 
-Open this URL `http://{host_ip}:5173` in your browser to access the svelte based frontend.
+4. Audio2Text Microservice
 
-Open this URL `http://{host_ip}:5174` in your browser to access the React based frontend.
+   ```bash
+    curl http://${host_ip}:9099/v1/audio/transcriptions \
+        -X POST \
+        -d '{"byte_str":"UklGRigAAABXQVZFZm10IBIAAAABAAEARKwAAIhYAQACABAAAABkYXRhAgAAAAEA"}' \
+        -H 'Content-Type: application/json'
+   ```
 
-### Svelte UI
+   Expected output:
 
-![project-screenshot](../../../../assets/img/docSum_ui_text.png)
+   ```bash
+     {"downstream_black_list":[],"id":"--> this will be different id number for each run <--","query":"you"}
+   ```
 
-### React UI (Optional)
-
-To access the React-based frontend, modify the UI service in the `compose.yaml` file. Replace `docsum-xeon-ui-server` service with the `docsum-xeon-react-ui-server` service as per the config below:
-
-```yaml
-docsum-xeon-react-ui-server:
-  image: ${REGISTRY:-opea}/docsum-react-ui:${TAG:-latest}
-  container_name: docsum-xeon-react-ui-server
-  depends_on:
-    - docsum-xeon-backend-server
-  ports:
-    - "5174:80"
-  environment:
-    - no_proxy=${no_proxy}
-    - https_proxy=${https_proxy}
-    - http_proxy=${http_proxy}
-  ipc: host
-  restart: always
-```
+5. Multimedia2text Microservice
 
-![preject-react-screenshot](../../../../assets/img/docsum-ui-react.png)
+   ```bash
+    curl http://${host_ip}:7079/v1/multimedia2text \
+        -X POST \
+        -d '{"audio":"UklGRigAAABXQVZFZm10IBIAAAABAAEARKwAAIhYAQACABAAAABkYXRhAgAAAAEA"}' \
+        -H 'Content-Type: application/json'
+   ```
+
+   Expected output:
+
+   ```bash
+     {"downstream_black_list":[],"id":"--> this will be different id number for each run <--","query":"you"}
+   ```
+
+6. MegaService
+
+   ```bash
+   curl -X POST http://${host_ip}:8888/v1/docsum \
```

Comment:
Please add the guideline for how to query with video/image/etc. with megaservice